items/weapons/primary.json

```
@@ -0,0 +1,272 @@
+{
+	"1": {
+		"name": "AR-23 Liberator",
+		"desc": "The SEAF standard assault rifle, balancing power, fire rate, and weight for a reliable weapon against smaller targets.",
+		"types": 1,
```

`types` implies plural, but I assume it consistently only has one type?
```suggestion
		"type": 1,
```